.github/workflows/integration-tests-on-release.yaml

```
@@ -0,0 +1,364 @@
+name: Integration tests on master/release
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+
+jobs:
+  TestCephFlexSuite-Master:
```

We don't need the `-Master` suffix on all the tests since this refactoring, right? Anyway they are run both on master and release branches.
```suggestion
  TestCephFlexSuite:
```